src/Utilities/Compiler/DiagnosticCategoryAndIdRanges.txt

```diff
@@ -17,7 +17,7 @@ Security: CA2100-CA2153, CA2300-CA2330, CA3000-CA3147, CA5300-CA5403
 Usage: CA1801, CA1806, CA1816, CA2200-CA2209, CA2211-CA2247
 Naming: CA1700-CA1726
 Maintainability: CA1500-CA1509
-Reliability: CA9999, CA2000-CA2015
+Reliability: CA9999, CA2000-CA2009, CA2011-CA2015
```

Comment:
Are we ok reusing CA2010 for something else in the future?

Reply:
Historically, we have avoided repurposing IDs given users might have existing suppressions/ruleset entries to turn these off. I am in two minds if we should make this change to indicate that we no longer use CA2010 OR keep the original range so we don't accidentally pick up CA2010. Maybe safer to go with the latter, let me revert this change.